webpack-demo/src/index.js

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line
```

Comment:
- [ ] Let's kindly fix :wrench: the Linter issue in our file. We are not supposed to disable Linters because they monitor our ways of coding and flag programming/stylistic errors. They are great tools and play an amazing role.